src/components/app.tsx

```diff
@@ -105,6 +105,14 @@ export const authAndConnect = async (stores: IStores) => {
       }
     }
 
+    // We don't want to connect to the database while we are authenticating the user in standalone mode.
+    // Once that finishes user.standaloneAuth will be undefined and this function will be called
+    // again to connect to the database
+    if (user.standaloneAuth) {
+      removeLoadingMessage("Connecting");
+      return;
+    }
+
```

Comment:
I don't understand how this comment relates to the code under it.

Reply:
The (badly named) `resolveAppMode` function connects to the database when the `appMode` is `"authed"`.  We switch the `"authed"` app mode in standalone mode at the end of the standalone auth.